public/docs/ts/latest/cookbook/production-deploy.jade

```diff
@@ -0,0 +1,327 @@
+  makes it easier than ever to use AoT and lazy-loading.
+  
+  You can use any build system you like; the only important thing is to optimize your build.
+  Whatever build system you choose make sure to automate it so that you can make a production
+  build in a single step.
+
+.l-main-section
+:marked
+  ## Angular configuration
+
+  ### The `base` tag
+
+  There is an important configuration item in the Angular Router that needs to be adjusted for 
+  different environments: 
+  [the `base` tags `href` property](https://angular.io/docs/ts/latest/guide/router.html#!#base-href).
+  This property tells the router what the app root URL is so that the router can match routes.
+
+  On your development server, your app is most likely served at the root: `http://localhost:3000/`.
+  Here you use `<base href="/">`, since `/` is the root of your app.
+  
+  But depending on the server setup that you use, the app can be served from a subfolder like
+  `http://localhost:3000/myapp/` or `http://www.mysite.com/myapp/`, where `/myapp/` is the subfolder.
+  In this case you should use `<base href="/myapp/">`, since `/myapp/` is the root of your app and 
+  not `/`.
+
+  When your `base` tag is misconfigured your browser console will show several errors in the console,
+  from missing files on wrong paths to router errors about missing routes.
+
+  ### enableProdMode
+
+  You can also configure your Angular app to start on 
+  [production mode](https://angular.io/docs/ts/latest/api/core/index/enableProdMode-function.html), 
+  which will make it faster by disabling development specific checks.
+
+  Angular apps default to development mode, as you can see by the following message on the browser
+  console:
+
+code-example(format="nocode").
+  Angular 2 is running in the development mode. Call enableProdMode() to enable the production mode.
+
+:marked
+  To enable production mode, add this code to the start of your `main.ts`:
+
+code-example().
+  import { enableProdMode } from '@angular/core';
+  enableProdMode();
+
+:marked
+  ### Lazy loading
+
+  The Angular Router allows you to configure NgModules as being 
+  [Lazy Loaded](https://angular.io/docs/ts/latest/guide/router.html#!#asynchronous-routing),
+  which means that particular NgModule (and all it's code) is not loaded on the very first load.
+  This allows your app to have a smaller initial load time by not loading modules until they
+  are needed.
+
+  A common mistake to make while lazy loading a module is to *also* have that module imported via
+  a ES6 import. 
+  If you do that, the module will never really be lazy loaded since the import statement will cause
+  it to be imported directly instead of being imported via the Router. 
+
+  It's important to note that your bundling configuration must take lazy loading into consideration.
+  Since there is no ES6 import for that module, bundlers don't know that they should also
```

Comment:
Perhaps we should just use `ES import` instead of `ES6` import. That safeguards us for future versions ... it is also what I do for all of my material :)